README.md

```diff
@@ -164,8 +164,8 @@ when you make a commit.
 
 ## Configuration
 
-terraform-docs can be configured with a yaml file. Default name of this file is
-`.terraform-docs.yml` and the order for lookig for it is:
+terraform-docs can be configured with a yaml file. The default name of this file is
```

Comment:
I like the new wording, and I think it might be good to update https://github.com/terraform-docs/terraform-docs/blob/master/docs/user-guide/configuration.md in similar way.

Reply:
@khos2ow I edited configuration.md per your suggestion and welcome your feedback